### PR TITLE
Devo theme

### PR DESCRIPTION
**Pull request by AlbertoCollado**
Devo Engineering Blog: Devo theme developed (alpha version)

### CODE REVIEW

File: _posts/2013-08-16-markup-syntax-highlighting.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+title: "Markup: Syntax Highlighting"
+excerpt: "Post displaying the various ways of highlighting code in Markdown."
+last_modified_at: 2018-01-03T09:45:06-05:00
+header:
+  teaser: "assets/images/devo-theme/hero_bg.jpg"
```
Comment (tripu) [inline _posts/2013-08-16-markup-syntax-highlighting.md]:
```suggestion
  teaser: "/assets/images/devo-theme/hero_bg.jpg"
```

**Review by tripu — CHANGES_REQUESTED** [review]
It looks great! Good job!

Just a suggestion.

The width available to actual content on a wide screen is kind of narrow, I think. If we were planning to share regular prose it wouldn't be a problem. But given that we will have snippets of code, commands, tables, diagrams, charts&hellip; it's a pity not being able to stretch things horizontally a bit more. I fear we will have horizontal scroll bars too often, when they should not be necessary.

Paradoxically, and because of the media queries, shrinking the window gets you to a point where the sidebar collapses and the `<article>` suddenly becomes _wider_. Right there it looks more legible to me.

I think that specific breakpoint is in line `1025` of `/assets/css/main.css` (not sure where's the actual source, though):

```css
@media (min-width: 64em) { .page { float: right; width: calc(100% - 200px); padding-right: 200px; } }
```

File: _posts/2013-08-16-markup-syntax-highlighting.md (hunk 1 above)
Comment (tripu) [reply _posts/2013-08-16-markup-syntax-highlighting.md]: (This is not important, because the post is just a placeholder, of course. FYI only.)

Comment (AlbertoCollado) [reply _posts/2013-08-16-markup-syntax-highlighting.md]: Thanks @tripu for reviewing ;) Regarding the content width issue, I have preferred to keep the blog's default implementation, bearing in mind that the default width in the big-screens break-point is what other sites like "Medium" usually use.

Also, we have to bear in mind that if you include: "classes: wide" in the props of your post, the content will automatically occupy all the available width.

Therefore, all you have to do is include this property in those posts that you want to occupy all the horizontal space available (look at the example post whose title is "" Wide Single Layout Page "")

Comment (tripu) [reply _posts/2013-08-16-markup-syntax-highlighting.md]: Ah! Great, I didn't realise of that. Perfect, then!

**Review by tripu — APPROVED** [review]
LGTM now!

**Pull request merged**